app/src/main/java/org/p2p/wallet/history/ui/history/HistoryFragment.kt

```diff
@@ -27,9 +30,15 @@ class HistoryFragment :
 
     override val presenter: HistoryContract.Presenter by inject()
     private val binding: FragmentHistoryBinding by viewBinding()
+
+    private var isRefreshing = false
     private val adapter: HistoryAdapter by unsafeLazy {
         HistoryAdapter(
-            onTransactionClicked = presenter::onItemClicked,
```

Comment:
I would suggest to remove kt-lint rule that we need to always put namedArg on a new line 
so instead of 
```kotlin
HistoryAdapter(
   onTransactionClicked = {}
)
```
we would have
```kotlin
HistoryAdapter(onTransactionClicked = {
    
})
```
no things to fix here, just a suggestion to discuss

Reply:
didn't understand why it's resolved, but ok...?